middleware/authenticate.js

```diff
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken');
+const secrets = require('../config/secrets');
+
+module.exports = () => {
+  return (req, res, next) => {
+    try {
+      const token = req.headers.authorization;
+      const decoded = jwt.verify(token, secrets.jwt);
+
+      req.userId = decoded.subject;
+      next();
+    } catch (err) {
+      return res.status(401).json({
+        message: 'User not authenticated. Please log in and try again.',
```

Comment:
Nothing logically wrong with your authenticate middleware, but on the post campaign routes this is the only thing stopping any user from posting a new campaign.  What would stop a regular supporter from posting a campaign?  What would stop a organization from posting a campaign to another organization?